### ISSUE TITLE
Choose the language, framework and tools

### ISSUE THREAD

**Issue by MedUnes** [MedUnes/100DaysOfCode#1]
I suggest a web application built with 

- Symfony2/MYSQL  as a backend
- Bootstrap/ JQUERY as a frontend

**Comment by AdventureBear** [comment]
I propose Node/Express/Mongo since this is a project stemming from my Free Code Camp studies.

**Comment by MedUnes** [comment]
@AdventureBear  Good argument that it is related to FCC so MEAN stack should be put in consideration.
Though, this raises up another problem, since finding a NodeJS server is harder/more expensive / less frequent than an LAMP (Linux Apache MYSQL PHP) server..

**Comment by AdventureBear** [comment]
What about Heroku ?

**Comment by MedUnes** [comment]
Good point, doesn't it put limitations on free accounts ?

**Comment by AdventureBear** [comment]
Yes, there are, but if the end goal in stuyding from FCC and using 100 days of code to do so is to learn how to create full stack web apps that scale, then its the best way to get started.  if this project scales to where we need to pay $7/month, we can ask for community donations.    otherwise it doesn't make sense to do it on a different tech stack, that's not the end goal (for me) of studying at FCC.

**Comment by MedUnes** [comment]
Nice, 7$ a month won't make a real problem anyway.
So I choose the MEAN stack.
Are you OK?

**Comment by AdventureBear** [comment]
I am still not well versed in the "A" or the "R" (MERN) stack, but learning React is my next project.  I'm fully on board with Mongo/Express/Node, and would need some guidance on Angular 2 vs React (or temporarily some simple front end such a EJS that could be replaced)

**Comment by MedUnes** [comment]
I guess FCC is rather with ReactJS, so if we want to get stick to the FCC learning plan,  we shall go for ReactJS, although I prefer Angular2 myself.
What do you think?

**Comment by H4wk-eye** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
colinbowen, @carlosf, @gerardma, @Alonski, @BunsDev, @jrodrigomg, @ghassheee, @listenaddress, @Zenedith, @Technetium1, @alephao, @bonneyinc, @AriaWang, @paddyzab, @hellokang, @tdevbb, @MHM5000, @entarrion, @NickSun, @gfredtech
